models/programdirector.js

```diff
@@ -14,8 +14,15 @@ module.exports = (sequelize, DataTypes) => {
     }
   };
   ProgramDirector.init({
-    directorType: DataTypes.STRING,
-    userOrgRole: DataTypes.STRING,
+    organizationName: DataTypes.STRING,
+    organizationType: DataTypes.STRING,
+    organizationPhoneNumber: DataTypes.INTEGER,
+    organizationAddress: DataTypes.STRING,
+    organizationCountry: DataTypes.STRING,
+    organizationState: DataTypes.STRING,
+    organizationCity: DataTypes.STRING,
+    organizationDescription: DataTypes.STRING,
+    partnershipDescription: DataTypes.STRING,
```

Comment:
Zizo, I see you've changed what is stored in the ProgramDirector model... which is a lot of info about the organization itself. Why put this here, instead of the Organization model? I think the original design was for ProgramDirector to be like a many-to-many join table, relating a User to an Organization in a particular role...